### PR TITLE
Allow strat execution from subdirectories

### PR DESCRIPTION
**Pull request by such**
Completes #115

### CODE REVIEW

File: strat/cmd/cmd.go [hunk 1]
```diff
@@ -150,8 +150,30 @@ func varsPath() string {
...
 
+func findProjectFile(wd string) (string, error) {
+	dir, err := filepath.Abs(wd)
+	if err != nil {
+		return "", err
+	}
+	for {
+		file := filepath.Join(dir, ProjectFile)
+		if _, err := os.Stat(file); os.IsNotExist(err) {
+			if dir == "/" {
```
Comment (stephan83) [inline strat/cmd/cmd.go]: Je me demande si ca marche sur Windows

Comment (stephan83) [inline strat/cmd/cmd.go]: > Abs calls Clean on the result.

> The returned path ends in a slash only if it represents a root directory, such as "/" on Unix or `C:\` on Windows.

Il y a donc juste a checker si ca se termine par / ou \ .

Comment (such) [inline strat/cmd/cmd.go]: I found it was cleaner to check if filepath.Dir(dir) == dir.
Let me know what you think.

Comment (stephan83) [inline strat/cmd/cmd.go]: Great

**Pull request closed without merging**